### ISSUE TITLE
Icons in different widths cause filenames to not be aligned

### ISSUE THREAD

**Issue by spatz** [jtbm37/all-the-icons-dired#10]
For example the dir icon is narrower than the emacs icon. Some sort of alignment should be added so that filenames all start in the same position.

**Comment by mohkale** [comment]
See [here](https://github.com/domtronn/all-the-icons.el/issues/77), the only real fix I can think of would be to add an option to customise the height of the icons. In my case, setting it all to 1.0 fixed the alignment issues.

**Comment by conao3** [comment]
Does [this PR](https://github.com/jtbm37/all-the-icons-dired/pull/31) solve this issue?

**Comment by mohkale** [comment]
@conao3 Yes, but the gap is a little large. Is there any way to customize that?

![Capture](https://user-images.githubusercontent.com/23294780/77155753-e998d480-6a95-11ea-838d-996eb51d8c5a.PNG)

**Comment by conao3** [comment]
What is `tab-width`? Is it `1`?

**Comment by mohkale** [comment]
nope, it was 4, just set it to 1 and it's working great. thnx as always 😄.

**Comment by tareefdev** [comment]
![image](https://user-images.githubusercontent.com/17391433/158448562-a122aa77-480c-4457-90e4-86af0923167e.png)

Is it fixed? I still see filenames are not aligned
